src/troubleshooting/fleet-commander.rst

```diff
@@ -0,0 +1,206 @@
+.. contents:: Table of Contents
+    :local:
+
+Troubleshooting Fleet Commander
+###############################
+
+This document contains common issues of Fleet Commander integration with SSSD.
+Please, refer to the `Fleet Commander`_ project documentation for general issues
+that are not related to the SSSD project. You can also read the
+:doc:`../design-pages/pages/fleet_commander_integration` design page to get a
+better understanding of the SSSD role and how the integration works under the
+hood.
+
+.. _Fleet Commander: https://fleet-commander.org
+
+How to get Fleet Commander's logs
+*********************************
+
+To enable verbose Fleet Commander's debug logs please set:
+
+.. code-block:: ini
+    :caption: /etc/xdg/fleet-commander-admin.conf
```

Comment:
Does a service need to be restarted for the changes to take effect?

Reply:
It is dbus activated so I don't think so.